kadai-4/src/pages/dashboard.vue

```diff
@@ -50,21 +92,58 @@ import store from '../store/index'
       },
     },
     methods: {
-      async logout(){
+      openModal: function(userName,wallet){
```

Comment:
他と統一させるために下記のようにしましょう。
```suggestion
      openModal(userName,wallet) {
```